projectsFunctions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
```

Comment:
is this file a duplicate of projectsCMSFunctions.php? if so get rid of one of them